scripts/update-gcloud-image/update-gcloud-image.service

```diff
@@ -3,6 +3,6 @@ Description=Check for update on KVM image.
 
 [Service]
 Type=oneshot
-User=gitlab-runner
-ExecStart=/bin/bash /etc/gitlab-runner/update-image.sh
+User=root
```

Comment:
You sure you want to run that script as root?

Reply:
I took that from the old machine :shrug: 

As far as I could see, the service was disabled in the old machine and you have been running this manually (locally?). How should we set it up? What is your advise?

Reply:
I added a note on [QA-480](https://tracker.mender.io/browse/QA-480) to review this (and enable it then)